recipe-generator/src/main/java/ca/derekcormier/recipe/generator/TypescriptIngredientGenerator.java

```diff
@@ -24,6 +24,11 @@ public void generate(Cookbook cookbook, String targetDir, Map<String, Object> op
         registerFilters(cookbook);
         String directory = createDirectories(targetDir);
 
+        if (!options.containsKey("ingredientPostfix")) {
```

Comment:
There is `putIfAbsent`, or can probably use `getOrDefault` a few lines down and remove this `if`.